.github/workflows/ci.yml

```diff
@@ -1,8 +1,8 @@
 on:
   push:
-    branches: ["master"]
+    branches: ["tokio-1.0.x"]
```

Comment:
Keep master as we will eventually merge into master. Also, we should be able to do a glob: "tokio-*.x".

Reply:
Example: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags